time  stamp

uint8 state

uint8 STATE_UNKNOWN     = 0
uint8 STATE_DISARMED    = 1
uint8 STATE_ARMED       = 2
uint8 STATE_MANUAL      = 3
uint8 STATE_TAKEOFF     = 4
uint8 STATE_LAND        = 5
uint8 STATE_RC_MODE     = 6
uint8 STATE_HOVER       = 7
uint8 STATE_GOTO        = 8
uint8 STATE_TRAJECTORY  = 9
